aten/src/ATen/native/TensorConversions.cpp

```diff
@@ -386,24 +387,31 @@ Tensor sparse_compressed_to_dense(
     dst_transposed.add_(to_transposed_csr);
     return dst_transposed.transpose(batch_ndim, batch_ndim + 1);
   }
-  if (self.layout() == kSparseBsr) {
-    auto crow_indices = self.crow_indices();
-    auto col_indices = self.col_indices();
+  if (self.layout() == kSparseBsr || self.layout() == kSparseBsc) {
+    Tensor compressed_indices;
+    Tensor plain_indices;
+    std::tie(compressed_indices, plain_indices) = [&]() {
+      if (self.layout() == kSparseBsr) {
+        return std::make_pair(self.crow_indices(), self.col_indices());
+      } else {
+        return std::make_pair(self.ccol_indices(), self.row_indices());
+      }
+    }();
```

Comment:
Might be a candidate for a utility function, with support for the other layouts?